lib/Runtime/Library/JavascriptSimdInt8x16.cpp

```diff
@@ -6,7 +6,9 @@
 
 namespace Js
 {
-    JavascriptSIMDInt8x16::JavascriptSIMDInt8x16(SIMDValue *val, StaticType *type) : JavascriptSIMDType(type), value(*val)
+    const char16 JavascriptSIMDInt8x16::_typeName[] = _u("SIMD.Int8x16");
+
+    JavascriptSIMDInt8x16::JavascriptSIMDInt8x16(SIMDValue *val, StaticType *type) : JavascriptSIMDType(val, type)
```

Comment:
Thank you, this was a missed during a previous refactoring. Added the type only constructors for the 3 simd types that were missing these.